hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -139,7 +130,6 @@ public ClusterServiceImpl(Node node) {
         clusterClock = new ClusterClockImpl(logger);
 
         thisAddress = node.getThisAddress();
-        thisMember = node.getLocalMember();
```

Comment:
Local member information can be changed before the start is completed. Therefore, it is not safe to cache local member here.